### PR TITLE
Text updates: ansible/ansible-tower-remediation

### PR DESCRIPTION
**Pull request by hillaryfraley**
Includes minor text revision in README, as well as these documented in the changelog:

- Update display_name to Ansible Tower Remediation (from Ansible Tower) to match GitHub
- Revise text labels and prompts in configuration modal
- Add pipeline reference setup step and example in README
- List and describe options for `io.sensu.ansible.config.actions` annotation in README
- Update README link references to match headings on the linked pages

### CODE REVIEW

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 1]
```diff
@@ -108,3 +143,4 @@ This integration does not produce any events that should be processed by an aler
 [ansible-tower-app-token]: https://docs.ansible.com/ansible-tower/latest/html/userguide/applications_auth.html#add-tokens
 [ansible-tower-api-reference]: https://docs.ansible.com/ansible-tower/latest/html/towerapi/api_ref.html
 [ansible-tower-auth-guide]: https://docs.ansible.com/ansible-tower/latest/html/administration/oauth2_token_auth.html
+[secret]:
```
Comment (hillaryfraley) [inline integrations/ansible/ansible-tower-remediation/README.md:146]:
```suggestion
[secret]: https://docs.sensu.io/sensu-go/latest/operations/manage-secrets/secrets/
```

File: integrations/ansible/ansible-tower-remediation/README.md [hunk 2]
```diff
@@ -59,37 +63,68 @@ This integration is compatible with the [Ansible Tower Jobs dashboard][ansible-t
          ]
    ```
 
+1. Add the `ansible-tower` pipeline to the same [checks].
```
Comment (hillaryfraley) [inline integrations/ansible/ansible-tower-remediation/README.md:66]:
```suggestion
1. Add the `ansible-tower` [pipeline] reference to the same [checks].
```

File: integrations/ansible/ansible-tower-remediation/sensu-integration.yaml [hunk 3]
```diff
@@ -102,14 +102,6 @@ spec:
...
-
-        ```yaml
-        spec:
-          pipelines:
-            - api_version: core/v2
-              type: Pipeline
-              name: ansible-tower
-        ```
\ No newline at end of file
+        Ansible Tower will process events from checks that include the the `io.sensu.ansible.config.actions` remediation action annotation _and_ `ansible-tower` pipeline on [[ ansible_tower_host ]].
```
Comment (hillaryfraley) [inline integrations/ansible/ansible-tower-remediation/sensu-integration.yaml:107]:
```suggestion
        Ansible Tower will process events from checks that include the the `io.sensu.ansible.config.actions` remediation action annotation _and_ `ansible-tower` pipeline on [[ ansible_tower_host ]].
        
        Add the `ansible-tower` pipeline reference to one or more checks:
        
         ```yaml
        spec:
          pipelines:
            - api_version: core/v2
              type: Pipeline
              name: ansible-tower
        ```
```

File: integrations/ansible/ansible-tower-remediation/sensu-integration.yaml [hunk 4]
```diff
@@ -102,11 +102,13 @@ spec:
       title: Success
     - type: markdown
       body: |
-        You have successfully enabled the Ansible Tower remediation integration.
+        Success: You enabled the Ansible Tower integration.
```
Comment (hillaryfraley) [inline integrations/ansible/ansible-tower-remediation/sensu-integration.yaml:105]:
```suggestion
        You enabled the Ansible Tower integration.
```

File: integrations/ansible/ansible-tower-remediation/sensu-integration.yaml [hunk 5]
```diff
@@ -102,11 +102,13 @@ spec:
       title: Success
     - type: markdown
       body: |
-        You have successfully enabled the Ansible Tower remediation integration.
+        You enabled the Ansible Tower integration.
```
Comment (hillaryfraley) [inline integrations/ansible/ansible-tower-remediation/sensu-integration.yaml:105]:
```suggestion
        You enabled the Ansible Tower Remediation integration.
```

**Review by jspaleta — APPROVED** [review]

**Comment by jspaleta** [comment]
Okay merging.  This should create a new issue with the need_tag label now. Once i get through a pile of these edit PRs today, I'll use the need_tag issues as a reference to add the tags

**Pull request merged**